samples/suit/recovery/prj.conf

```diff
@@ -0,0 +1,101 @@
+# #
+# # Copyright (c) 2024 Nordic Semiconductor ASA
+# #
+# # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+# #
+
+CONFIG_SUIT_MPI_GENERATE=n
+CONFIG_SUIT_ENVELOPE_OUTPUT_MPI_MERGE=n
+
+CONFIG_SUIT_ENVELOPE_TEMPLATE=""
```

Comment:
You can use `CONFIG_SUIT_LOCAL_ENVELOPE_GENERATE` instead.

Reply:
Changed